src/UuidInterface.php

```diff
@@ -129,7 +131,7 @@ public function getDateTime(): DateTimeInterface;
     /**
      * Returns the 128-bit integer value of the UUID as a string
      *
-     * @return mixed
+     * @return string
```

Comment:
This happened to be fixed during static analysis runs.

@psalm prevents usage of `mixed` (good) when configured with `totallyTyped="true"`: this was probably left un-documented to avoid excluding the `SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint` rule from CS standards configuration (see change in `phpcs.xml.dist` in this patch)